src/index.ts

```diff
@@ -12,7 +12,7 @@ import {
   createRollupPrepareCustomFeeTokenApprovalTransactionRequest,
   CreateRollupPrepareCustomFeeTokenApprovalTransactionRequestParams,
 } from './createRollupPrepareCustomFeeTokenApprovalTransactionRequest';
-import { CreateRollupFunctionInputs, CreateRollupParams } from './types/createRollupTypes';
+import { CreateRollupFunctionInputs, CreateRollupTxParams } from './types/createRollupTypes';
```

Comment:
I get the point of the changes, but because it's exported to consumers, we might not want to change it? Or maybe rename it before we export?

It's just a type, so it might be overkill